ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+is invoked.
+
+As stated above the ivocation fails. This failure simulates a intermittent network error
+at time the transaction two-phase commit protocol
+already decided that the work has to be committed. The observer is not informed
+about the intermittent failure as it's responsibility of recovery manager to finish
+all the work.
+
+The work, which has to be finished by recovery, is consisted of committing two XAResources
+which were part of the business method at the caller side of artifact `server.war`.
+First is data insertion to a database. Second is a testing XAResource which does
+no real work but it's capable to inform us if it was committed.
+You can ask the server about the number succesful of commits of the testing XAResource
+by invoking REST endpoint `http://localhost:8180/server/commits`.
+
+The http://jbossts.blogspot.com/2018/01/narayana-periodic-recovery-of-xa.html[recovery manager]
+normally executes the recovery processing in periodically every 2 minutes.
+When the recovery process is started the resources at the remote server (on the callee side)
+are committed.
+
+You may speed up the process and invoke the recovery process manually by accessing
+the port where recovery manager listener listens at. The recovery listener was enabled
+for this purpose by cli command, see `${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/remote-configuration.cli`.
+For starting the recovery processing send `SCAN` command to socket at `localhost:4712`.
+
+[source]
+----
+telnet localhost 4712
+Trying 127.0.0.1...
+Connected to localhost.
+Escape character is '^]'.
+SCAN
+DONE
+Connection closed by foreign host.
+----
+
+Steps for observe the processing
+
+. Invoke the endpoint
++
+[source,options="nowrap"]
+----
+curl -s http://localhost:8080/client/remote-outbound-fail-stateless
+----
++
+. Check the server logs
+.. The `server1` will contain error
++
+[source,options="nowrap"]
+----
+ARJUNA016036: commit on < formatId=131077, gtrid_length=35, bqual_length=36, tx_uid=..., node_name=server1, branch_uid=..., subordinatenodename=null, eis_name=unknown eis name > (Subordinate XAResource at remote+http://localhost:8180) failed with exception $XAException.XA_RETRY: javax.transaction.xa.XAException: WFTXN0029: The peer threw an XA exception
+----
++
+. The `server2` or `server3` will contain error
++
+[source,options="nowrap"]
+----
+ARJUNA016036: commit on < formatId=131077, gtrid_length=35, bqual_length=43, tx_uid=..., node_name=server1, branch_uid=..., subordinatenodename=server2, eis_name=unknown eis name > (org.jboss.as.quickstarts.ejb.mock.MockXAResource@731ae22) failed with exception $XAException.XAER_RMFAIL: javax.transaction.xa.XAException
+----
++
+. Verify number of commits done for the test XAResource at the server side
++
+[source,options="nowrap"]
+----
+curl -s http://localhost:8180/server/commits
+curl -s http://localhost:8280/server/commits
+----
```

Comment:
What does the output mean?
What are the expected results?
How can decide whether the expected behaviour happened (I need this information in order to decide whether the quickstart is operating correctly)?

Reply:
I changed the text, hopefully it's now more meaningful.